docusaurus/docs/operate/configs/relayminer_config.md

```diff
@@ -451,3 +457,44 @@ flowchart
 ## Supported server types
 
 The list of supported server types can be found at [pkg/relayer/config/types.go](https://github.com/pokt-network/poktroll/tree/main/pkg/relayer/config/types.go#L8)
+
+## Payable Proof Submissions
+
+### Overview
+
+Submitting `Proofs` requires the payment of a fee, which is defined by the
+`proof_submission_fee` governance parameter.
+
+This fee mechanism is designed to prevent spam attacks (e.g., Sybil bloat attacks)
+by discouraging invalid or unnecessary `Proofs`.
+
+Since `Proofs` are the most block space-consuming primitive in the Pocket Network,
+this fee plays a crucial role in maintaining the integrity of the network.
+
+### Key Requirements for Operators
+
+Due to the probabilistic nature of the `Proof` requirement, `Supplier` operators must
+ensure that they maintain sufficient funds to cover the submission fees for all
+`Proof`s associated with the `Claim`s they plan to submit.
+
+Failure to maintain adequate funds can result in missed submissions, which can
+result in `Supplier` slashing if the `Proof` is required.
+
+### Recommendations for Supplier Operators
+
+- **Sufficient Balance**: Operators should regularly check their account balance
+and ensure they have enough funds to cover the submission fees for `Proofs`.
+
+- **Automated Monitoring**: It is highly recommended to implement balance monitoring
+or alert systems to avoid running out of funds during critical periods.
+
+- **Cost Awareness**: Operators should familiarize themselves with the current
+`proof_submission_fee` and plan their funding accordingly, especially if they
+anticipate submitting a higher volume of `Proofs`.
+
+### Implications for Network Participation
+
+Having insufficient funds could lead to rejected `Proof` submissions and
+potentially disrupt the operator’s participation in the Pocket Network.
+To maintain a smooth operation, operators must plan and manage their account
+balance as part of their operational procedures.
```

Comment:
```suggestion
:::warning

If you submit a claim but do not submit a proof, this could result in SLASHING!

Having insufficient funds could lead to rejected `Proof` submissions . This
can disrupt the operator’s participation in the Pocket Network. To maintain a
smooth operation, avoid slashing, and earn your rewards, operators must plan
and manage their account balance as part of their operational procedures.

:::
```